docs/user/dashboard/dashboard.asciidoc

```diff
@@ -337,6 +337,8 @@ Apply a set of design options to the entire dashboard.
 
 * *Sync color pallettes across panels* &mdash; Applies the same color palette to all panels on the dashboard.
 
+* *Sync tooltips across panels* &mdash; When you hover your cursor over a chart, the tooltips on all other related dashboard charts automatically appear.
```

Comment:
Just a question: Should we be more specific here about the charts that support this feature? For example this won't be applied to the partition charts.

Reply:
Actually it is supported only for the XY charts in Lens, Timelion, TSVB and agg-based visualizations